controllers/chat/getSignedChannelList.js

```diff
@@ -62,7 +62,7 @@ const __queryChannelBuilder = async (client, userId, limit, offset) => {
  * @param {import('stream-chat').Channel} channel
  * @returns
  */
-const __filterAndTransformChannelData = async (channel) => {
+const __filterAndTransformChannelData = async (channel, userId) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `__filterAndTransformChannelData` now takes an additional parameter `userId`. Ensure that all calls to this function have been updated to pass this new argument.
```diff
- const __filterAndTransformChannelData = async (channel) => {
+ const __filterAndTransformChannelData = async (channel, userId) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->